app/helpers/invoices_helper.rb

```diff
@@ -27,4 +27,8 @@ def get_taxes_parent
   def get_taxes_data
     Tax.all
   end
+
+  def get_invoice(id)
```

Comment:
@PragatiDoshi-BTC The name should be `get_invoice_by_id` but mostly such methods are used in controller to load DB row using `params[:id]`